include/aspect/material_model/rheology/visco_plastic.h

```diff
@@ -114,6 +114,8 @@ namespace aspect
                                             const std::vector<double> &volume_fractions,
                                             const std::vector<double> &phase_function_values = std::vector<double>(),
                                             const std::vector<unsigned int> &n_phases_per_composition =
+                                              std::vector<unsigned int>(),
+                                            const std::vector<unsigned int> &fluid_pressures =
```

Comment:
Why a vector? Is there more than one type of fluid pressure we may want to pass over? Or will this variable also be used to store the porosity?